content/livesync/models/models.textile

```diff
@@ -66,21 +65,21 @@ const modelsClient = modelsClient.models.get({
 });
 ```
 
-Explicitly call the sync function directly on the model in order to bootstrap its state on initial page load, for example:
+Call the sync function directly on the model to bootstrap its state during the initial page load:
 
 ```[javascript]
 await model.sync(123, 1);
 ```
 
-If calling the sync function throws an error, the Models SDK will automatically retry according to the strategy specified by the @syncOptions.retryStrategy@ defined in your "@ClientOptions@":/livesync/models#options.
+The SDK automatically attempts to re-execute the sync function in the case of an error, as specified by the @syncOptions.retryStrategy@ defined in your "@ClientOptions@":/livesync/models#options.
 
 h3(#sequence). SequenceID
 
-The @sequenceID@ enables the Models SDK to know the point in the stream of change events that corresponds to this version of the database state.
+The @sequenceID@ enables the SDK to identify the point in the stream of change events that corresponds to the current version of the database's state.
 
-Internally, the Models SDK will replay change events from the point specified by the @sequenceID@ onwards in order to bring the model state up to date with any realtime changes that occur from the time at which the state was read from the database.
+Internally, the SDK replays change events starting from the point indicated by the @sequenceID@ to update the model state with any realtime changes occurring since the state was read from the database. 
 
-Your backend endpoint called by your sync function should return the largest @sequenceID@ in the "outbox":/livesync/connector/database#outbox the time that the state is read:
+The sync function should call your backend endpoint to return the largest @sequenceID@ present in the outbox when reading the state:
```

Comment:
Thanks Greg 👍

Reply:
Please see d517d46f2e3a834a597d19028d21539d080bb67b

Reply:
I think the important point here is that the customer's backend endpoint return the sequence ID, but this change seems to make the sync function the subject of the sentence instead of the endpoint